idea-plugin/src/main/kotlin/org/jetbrains/compose/desktop/ide/preview/RunPreviewAction.kt

```diff
@@ -14,16 +14,20 @@ import com.intellij.openapi.externalSystem.service.execution.ProgressExecutionMo
 import com.intellij.openapi.externalSystem.task.TaskCallback
 import com.intellij.openapi.externalSystem.util.ExternalSystemUtil.runTask
 import com.intellij.openapi.util.UserDataHolderBase
+import com.intellij.openapi.wm.ToolWindowManager
 import org.jetbrains.plugins.gradle.settings.GradleSettings
 import org.jetbrains.plugins.gradle.util.GradleConstants
-import java.util.concurrent.CompletableFuture
+import javax.swing.SwingUtilities
 
 class RunPreviewAction(
     private val fqName: String,
     private val modulePath: String
 ) : AnAction({ "Show non-interactive preview" }, PreviewIcons.RUN_PREVIEW) {
     override fun actionPerformed(e: AnActionEvent) {
         val project = e.project!!
+        val previewToolWindow = ToolWindowManager.getInstance(project).getToolWindow("Desktop Preview")
+        previewToolWindow?.setAvailable(true)
```

Comment:
Maybe we call it in `onSuccess`? Until then preview will be hidden and user will se running action in the statusbar